### PR TITLE
fix: make trigger modification in different file

### PR DESCRIPTION
**Pull request by NGA-TRAN**
Fix migration

- [ ] I've read the contributing section of the project [README](https://github.com/influxdata/influxdb_iox/blob/main/README.md).
- [ ] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed).

### CODE REVIEW

File: iox_catalog/migrations/20230105120822_trigger_update_partition.sql [hunk 1]
```diff
@@ -7,7 +7,7 @@ RETURNS TRIGGER
 LANGUAGE PLPGSQL
 AS $$
 BEGIN 
-    UPDATE partition SET new_file_at = NEW.created_at WHERE id = NEW.partition_id;
+    UPDATE partition SET new_file_at = EXTRACT(EPOCH FROM now() ) * 1000000000 WHERE id = NEW.partition_id;
```
Comment (NGA-TRAN) [inline iox_catalog/migrations/20230105120822_trigger_update_partition.sql:10]: Convert this back to its original content

File: iox_catalog/migrations/20230106145820_modify_trigger.sql [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+CREATE OR REPLACE FUNCTION update_partition_on_new_file_at()
+RETURNS TRIGGER 
+LANGUAGE PLPGSQL
+AS $$
+BEGIN 
+    UPDATE partition SET new_file_at = NEW.created_at WHERE id = NEW.partition_id;
+
+    RETURN NEW;
+END;
+$$;
```
Comment (NGA-TRAN) [inline iox_catalog/migrations/20230106145820_modify_trigger.sql:11]: Make the modification in a newer file

Comment (NGA-TRAN) [reply iox_catalog/migrations/20230106145820_modify_trigger.sql:11]: I have run end-to-end and iox-catalog tests with postgres option and they work. Hopefully, this I the right fix for the migration

**Review by pauldix — APPROVED** [review]
Ok, I think this will do it, assuming the change brings it back to exactly what the old file was.

**Pull request merged**